README.md

```diff
@@ -167,5 +189,18 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+
+1. genie = Genie.create(name: "Genie")
+good_lamp = Lamp.create(wishes_remaining: 3)
+
+2. genie = good_lamp.genies.update(name: "Genie")
+
+3. good_lamp.update(wishes_remaining: 1)
+
+4. jafar = Genie.create(name: "Jafar")
+evil_lamp = Lamp.create(wishes_remaining: 3)
+jafar = evil_lamp.genies.update(name: "Jafar")
+
+5. good_lamp.update(wishes_remaining: nil)
+
```

Comment:
Since a lamp has only one genie, it would be `good_lamp.genie.update(name: "Genie")`.

Also, the above would change the genie's name. To actually associate the lamp and the genie, you'd need:

```
juan = Genie.create(name: "Genie")
good_lamp = Lamp.create(wishes_remaining: 3)
good_lamp.update(genie: juan)
```